src/current/_includes/v24.3/sidebar-data/self-hosted-deployments.json

```diff
@@ -406,89 +406,95 @@
                   "title": "Metrics Dashboards",
```

Comment:
Nice! I like this change to the nav! This is a good reminder because I would like to simplify some of the navs in dr/cdc